randovania/data/json_data/prime1.txt

```diff
@@ -1292,7 +1267,7 @@ Asset id: 4157096768
 Security Cave
 Asset id: 1016369381
```

Comment:
Security cave, add morph to both nodes requirements

Reply:
I did that bit in the one where I fixed quarantine monitor.